### PR TITLE
Adding show details button for applications in referent space

### PR DESCRIPTION
**Pull request by ktherage**

### CODE REVIEW

File: src/Controller/EnMarche/ReferentController.php [hunk 1]
```diff
@@ -645,10 +647,52 @@ public function municipalVolunteerAction(
...
+     *     name="app_referent_municipal_running_mate_request_detail",
+     *     requirements={
+     *         "uuid": "%pattern_uuid%",
+     *     },
+     *     methods={"GET"},
+     * )
+     *
+     * @Security("is_granted('ROLE_REFERENT')")
+     *
+     * @Entity("runningMateRequest", expr="repository.findOneByUuid(uuid)")
```
Comment (ottaviano) [inline src/Controller/EnMarche/ReferentController.php]: is it not working without? 🤔

File: src/Referent/MunicipalExporter.php [hunk 2]
```diff
@@ -44,6 +44,12 @@ public function exportRunningMateAsJson(array $runningMates): string
                     ]),
                 ],
                 'isAdherent' => $runningMate->isAdherent() ? 'Oui' : 'Non',
+                'detailLink' => [
+                    'label' => "<span id='survey-edit-$i' class='btn btn--default'>Voir le détail</span>",
```
Comment (ottaviano) [inline src/Referent/MunicipalExporter.php]: why do we need to have this id `survey-edit-$i`?

File: templates/referent/municipal/running_mate/detail.html.twig [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{% extends 'referent/_layout.html.twig' %}
+
+{% block referent_content %}
+    <a href="{{ path('app_referent_municipal_running_mate_request') }}" class="back-to-list icon--with-text b__nudge--top-15 b__nudge--bottom">
+        {{ include('components/caret--left.html.twig') }}
+        Retour à la liste des candidats colisiters
```
Comment (ottaviano) [inline templates/referent/municipal/running_mate/detail.html.twig]: `colistiers`*

File: src/Controller/EnMarche/ReferentController.php (hunk 1 above)
Comment (ktherage) [reply src/Controller/EnMarche/ReferentController.php]: well, it seems to work. I thought this was mandatory to get it working.

File: templates/referent/municipal/running_mate/detail.html.twig [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        {{ include('components/caret--left.html.twig') }}
+        Retour à la liste des candidats colistiers
+    </a>
+
+    <hr />
+
+    <div class="l__wrapper--slim">
+        <strong>Nom :</strong> {{ runningMateRequest.lastName }}<br>
+        <strong>Prénom :</strong> {{ runningMateRequest.firstName }}<br>
+        <strong>Téléphone :</strong> {{ runningMateRequest.phone ? runningMateRequest.phone | phone_number_format : '' }}<br>
```
Comment (thomasbisignani) [inline templates/referent/municipal/running_mate/detail.html.twig]: Can you remove the whitespaces on the twig filters please ?

`runningMateRequest.phone | phone_number_format`

 =>

`runningMateRequest.phone|phone_number_format`

File: templates/referent/municipal/volunteer/detail.html.twig [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+        <strong>Prénom :</strong> {{ volunteerRequest.firstName }}<br>
+        <strong>Téléphone :</strong> {{ volunteerRequest.phone ? volunteerRequest.phone | phone_number_format : '' }}<br>
+        <strong>Profession :</strong> {{ volunteerRequest.profession }}<br>
+        <strong>Thèmes favoris :</strong>
+        <ul>
+        {% for favoriteTheme in volunteerRequest.favoriteThemes %}
+            <li>{{ favoriteTheme.name }}</li>
+        {% endfor %}
+        </ul><br>
+        <strong>Thèmes favoris personnalisé :</strong><blockquote>{{ volunteerRequest.customFavoriteTheme }}</blockquote><br>
```
Comment (thomasbisignani) [inline templates/referent/municipal/volunteer/detail.html.twig]: `Thèmes favoris personnalisé` => `Thèmes favoris personnalisés`

**Comment by thomasbisignani** [comment]
Can you add tests for this new part ?

**Comment by ktherage** [comment]
> Can you add tests for this new part ?

Done.

**Pull request merged**